kubejs/server_scripts/enigmatica/kubejs/base/recipetypes/thermal/machine/insolator_catalyst.js

```diff
@@ -7,33 +9,38 @@ onEvent('recipes', (event) => {
                 secondaryMod: 2.0,
                 energyMod: 0.8,
                 minChance: 0.8,
-                useChance: 0.8
+                useChance: 0.8,
+                id: `${id_prefix}fertilizer`
             },
             {
                 input: 'botania:fertilizer',
                 primaryMod: 1.7,
                 secondaryMod: 1.7,
                 energyMod: 0.9,
                 minChance: 0.5,
-                useChance: 0.5
+                useChance: 0.5,
+                id: `${id_prefix}floral_fertilizer`
             },
             {
-                input: 'farmingforblockheads:red_fertilizer',
+                input: 'farmingforblockheads:red_fertilizer', // no green fertilizer here too?
```

Comment:
no clue 😅

Reply:
afterwards i read something ingame about one color speeding growth and another increasing yield, i suppose one of them is not supported in here but i recalled both green & red existing since i came across them elsewhere 🤔